### DISCUSSION TITLE
Some components do not work on Windows 11 22631.2361

### DISCUSSION THREAD

**Discussion by SheepMoods** [valinet/ExplorerPatcher#2196] [General]
Whether it's an update installation or aan upgrade installation of Windows 11 23H2, EP can't download the symbol file. Clicking the Start button does not bring up the Start menu. The build version is 22631.2361.

Here's the video.
https://github.com/valinet/ExplorerPatcher/assets/28479262/2627941d-9ddb-4172-b4e0-827a9289a637

**Comment by Amrsatrio** [comment]
Symbols are not available for Release Preview builds, but I've made stuff that makes Win10 taskbar Win+X and Win10 Alt+Tab work. The start menu should be able to show up if you were connected to the Internet when installing EP.

Please try to reinstall EP **while being connected to the internet** because it downloads files responsible to make the old start menu work. If they still don't work, please install the latest prerelease version. Let me know how it goes.

**Reply by SheepMoods** [reply]
I don't plan on switching back to Start10. I'm installing EP mainly to use the old Explorer back and to turn off that recommended item in the Start menu. If EP is only for GA I can wait until next month's update to install it.

**Reply by pyrates999** [reply]
In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

**Reply by Amrsatrio** [reply]
If you are talking about customizing the Win11 start menu, "Disable the Recommended section" is not _yet_ supported on 23H2 builds. I just upgraded to 2361, and turns out symbols are available for it, though. That's why "Open Start in All apps by default" works fine. But I will leave this issue to Valinet who knows how to get around this. 

Due to limited free time on my end, I'm only interested in making patches to fix major functions of the patcher.

You may create a new issue that accurately describes this so should we want to support this in the future, we can refer to your issue.

**Comment by ghost** [comment]
I had this problem just now. Every update since one of the Spring 2023 Windows Updates has made EP unable to download the OS symbols. I end up having to reset all settings to default, uninstall EP, reinstall EP, fiddle around with the OS symbols download settings, rinse and repeat until it finally gets the update.

When I finally got the new symbols, the start menu would not show, so I uninstalled EP again, reinstalled it again, and now I have Windows 10 start menu back again.

So, you may have to go around the carnival carousel a few times before it starts to work properly.

**Reply by Amrsatrio** [reply]
Yeah. This is why I tried to reduce the reliance on symbols. But regarding the start menu requiring Internet connection during installation of EP and the fact that the user doesn't get notified when start menu files couldn't be downloaded, I'll leave that to Valinet.

**Comment by Anixx** [comment]
Can you make the non-immersive taskbar menus also to work without symbols?

**Reply by SheepMoods** [reply]
"Non-immersive taskbar menus" has always been included in the system and is there whenever you switch to Taskbar10. I'm not quite sure what the symbol file does though.

**Reply by Anixx** [reply]
It des not work without symbols

**Comment by SheepMoods** [comment]
Exciting. I tried it again. Some of the components that have been modified still work correctly if upgrading from 22621.2361, where EP is already installed, to 22631.2361 (just installing "Microsoft-Windows-23H2Enablement-Package").

**Reply by SheepMoods** [reply]
Except Start10 is not working. Here are some screenshots.

<img width="840" alt="Screenshot 2023-10-01 123310" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/10f1b2ab-4651-41e2-8922-4a4463bb4915">
<img width="840" alt="Screenshot 2023-10-01 122608" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/c2f0c4a8-95d7-4a8e-9f16-7aa19d994fa1">
<img width="840" alt="Screenshot 2023-10-01 122413" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/f923a004-d92d-4271-a9d0-836112724629">
<img width="840" alt="Screenshot 2023-10-01 122452" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/dfee11d1-7f43-4126-ab22-61074fac8222">
<img width="840" alt="Screenshot 2023-10-01 122525" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/f3586f6c-87f4-43ca-b7ff-704bf0f9f8ba">
<img width="840" alt="Screenshot 2023-10-01 122905" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/1f9ea4be-4af4-4218-a424-c8eba112f20d">
<img width="840" alt="Screenshot 2023-10-01 124727" src="https://github.com/valinet/ExplorerPatcher/assets/28479262/d1c7dc00-3c07-48bc-96d3-36c57c5a63e0">
